aiohttp/web.py

```diff
@@ -27,7 +27,7 @@
 from .web_protocol import *  # noqa
 from .web_request import *  # noqa
 from .web_response import *  # noqa
-from .web_server import Server
+from .web_server import Server as WebServer
```

Comment:
Maybe rename web_server.Server into web_server.WebServer to avoid collisions?

Reply:
This will break retro compatibility but we can maybe

Reply:
Actually I just saw this in a test traceback:

```
>       srv = web.Server(handler)
E       TypeError: __init__() takes 1 positional argument but 2 were given
```

Which means the name Server is exported in `aiohttp.web` so I need to find another name than `Server` to avoid breaking compatibility.

Reply:
Yea, better to avoid this name collision somehow.